src/ZODB/scripts/repozo.py

```diff
@@ -730,6 +776,8 @@ def main(argv=None):
             do_backup(options)
         except WouldOverwriteFiles as e:
             sys.exit(str(e))
+        except VerificationFail as e:
+            sys.exit(str(e))
```

Comment:
Here do you propose to use base class for all 3 repozo custom errors (WouldOverwriteFiles, NoFiles and VerificationFail), or do you propose to merge some of them ? From my understanding, all these exceptions are not raised because of bugs, but just to inform user that repozo didn't finish correctly, most likely because of misuse or protection of users' data.

Reply:
Yes, a base class for the three custom error classes, with one `except ... as e: sys.exit(str(e))` clause to catch them all in the main function.

I would suggest a name if I could come up with one ;)

Error?  Failure?